Gruntfile.js

```diff
@@ -362,12 +362,18 @@ module.exports = function(grunt) {
   grunt.registerTask("dist-compile", ["test", "uglify", "compress"]);
 
   grunt.registerTask("commitjs", ["dist-compile", "gitcommit:built"]);
-  grunt.registerTask("default", ["connect", "dev-compile", "watch"]);
+  grunt.registerTask("default", ["connect", "dev-compile", "watch-silent"]);
 
   grunt.registerTask("test", ["dev-compile", "test-local"]);
   grunt.registerTask("test-local", ["blanket_mocha", "ts:verifyDefinitionFiles", "lint"]);
   grunt.registerTask("test-sauce", ["connect", "saucelabs-mocha"]);
 
+  grunt.registerTask("watch-silent", function() {
+    // Surpresses the "Running 'foo' task" messages. More info at #2507.
+    grunt.log.header = function() {};
```

Comment:
Hmm, pointing at the issue I don't think reveals any more information.  I think if you want to put a comment there, I would just put the first part of the comment.  I think the "More info..." is unnecessary, especially if this will close #2507